src/frontend/src/App.module.scss

```diff
@@ -4,4 +4,8 @@
   height: 100vh;
   flex-flow: column wrap;
   justify-content: center;
+  * {
+    font-family: Open Sans, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
+      Helvetica Neue, Helvetica, Arial, sans-serif !important;
```

Comment:
what is `!important` used for?

Reply:
Great question! So `!important` is to ensure that the rule will take precedence over other stylesheets that are setting the `font-family` rule. This is needed because we still have Semantic UI css stylesheet, which sets `font-family` to a different font.

And we still need Semantic UI, because we have components using Semantic UI. On top of that, when we get desperate we might need to port IDseq's components directly (which are using Semantic UI) instead of rewriting them in Material UI. 

However, all new styling uses Material UI by default now, so we will rely on Semantic UI as minimally as possible!